webclient/core/urls.py

```diff
@@ -22,7 +22,10 @@
         name="upload_file",
     ),
     path("soubor/stahnout/<str:typ_vazby>/<str:ident_cely>/<int:pk>", views.DownloadFile.as_view(), name="download_file"),
-    path("soubor/stahnout-nahled/<str:typ_vazby>/<str:ident_cely>/<int:pk>", views.DownloadThumbnail.as_view(), name="download_thumbnail"),
+    path("soubor/stahnout-nahled/<str:typ_vazby>/<str:ident_cely>/<int:pk>", views.DownloadThumbnailSmall.as_view(),
+         name="download_thumbnail"),
+    path("soubor/stahnout-nahled-velky/<str:typ_vazby>/<str:ident_cely>/<int:pk>",
```

Comment:
@pesikj @jnihnat Toto se nám prosím ještě musí dostat také do oprávnění. Řádky v tabulce URLs jsem připravil. Obecně - kdykoli přidáváme upravujeme URL, musíme to z hlediska oprávnění podchytit. Prosím, myslete na to.

Reply:
mnelo by stacit upravit excel a naimportovat. Obecne pokud pridame url a neni tam neco specialniho malo by to stacit upravit excel a naimportovat.

Reply:
Je to tam, víceméně jako kopie URL pro malé náhledy. Jen nevím, zda nějak není třeba ošetřit ty názvy akcí.